### ISSUE TITLE
Bundle Pool Cleanup Agent makes Eclipse un-launchable

### ISSUE THREAD

**Issue by nlisker** [eclipse-oomph/oomph#109]
Installer version: 1.34.0 Build 753

After installing Eclipse Platform 4.33 with bundles enabled, Eclipse can be launched. If you then go to Bundle Pools > Cleanup Agent and clean "unnecessary" items, Eclipse doesn't launch anymore with the message:

![image](https://github.com/user-attachments/assets/1d4cc2e8-c308-46f3-a4fa-bc9a9f50e890)

The cleaner is cleaning too much apparently.

**Comment by merks** [comment]
Yes, it's not really been tested in a very long time and I'd personally be a bit (very) scared to use it on my pool that supports around 100 installations.   It's not an easy thing to test and debug either.  Sorry about that. 😢

**Comment by nlisker** [comment]
Perhaps disable it then for the meantime.

**Comment by merks** [comment]
Yes, or at the very least, a big warning, warning, warning to create a backup in the likely case of overly aggressive removals.  I'll try to find some time for testing and failing to find an obvious problem that can be fixed, take some other remedial action, e.g., creating a backup folder with the deleted content.  Not sure...

**Comment by mmosconi1** [comment]
> Yes, it's not really been tested in a very long time and I'd personally be a bit (very) scared to use it on my pool ...

That worries me a bit... 
I'm using it frequently (without problems so far), mainly to get rid of profiles of deleted workspaces that I want  to re-create under the same path/name.

**Comment by merks** [comment]
I did test it since I wrote my comment and it worked fine for me, on a test user folder.   So I'm not really sure what problem there might be...
